library/modules/bigip_virtual_server.py

```diff
@@ -1747,12 +1758,20 @@ def _handle_profile_context(self, tmp):
         tmp['context'] = tmp['context'].replace('client-side', 'clientside')
 
     def _handle_ssl_profile_nuances(self, profile):
-        if profile['name'] == 'serverssl' or self._is_server_ssl_profile(profile):
-            if profile['context'] != 'serverside':
-                profile['context'] = 'serverside'
-        if profile['name'] == 'clientssl' or self._is_client_ssl_profile(profile):
-            if profile['context'] != 'clientside':
-                profile['context'] = 'clientside'
+        if self._values['check_profiles']:
```

Comment:
i was under the impression we are disabling ALL of the checks with this not just profiles?

Reply:
I don't think we need to, or should. The offending check is rather long and cumbersome. So I think we should start from there. If customers see a performance improvement that's enough, we should stick with it. Else we can expand it.

I also had the option to add the check to each individual profile, but that didn't make sense.